apps-rendering/src/components/Byline/CommentByline.tsx

```diff
@@ -0,0 +1,47 @@
+import type { SerializedStyles } from '@emotion/react';
+import { css } from '@emotion/react';
+import { text } from '@guardian/common-rendering/src/editorialPalette';
+import type { ArticleFormat } from '@guardian/libs';
+import { headline } from '@guardian/source-foundations';
+import type { Option } from '@guardian/types';
+import { darkModeCss } from 'styles';
+import { getThemeStyles } from 'themeStyles';
+import { DefaultByline } from './Byline.defaults';
+
+const commentStyles = (kicker: string): SerializedStyles => css`
+	color: ${kicker};
+	width: 75%;
+	${headline.medium({ fontWeight: 'light', fontStyle: 'italic' })}
+`;
+
+const commentAnchorStyles = (
```

Comment:
Nitpick, but in this file can these just be `styles` and `anchorStyles`? Slightly fewer terms to hold in your head 🤔